src/main/java/apoc/coll/Coll.java

```diff
@@ -431,6 +431,30 @@ public boolean containsAllSorted(@Name("coll") List<Object> coll, @Name("values"
         return true;
     }
 
+    @UserFunction
+    @Description("apoc.coll.isEqualCollection(coll, values) return true if two collections contain the same elements with the same cardinality in any order (using a HashMap)")
```

Comment:
Can we simplify this? The implementation is quite complex, I think that something like this should do the work:
```
        Map<Object, Long> map1 = first.stream()
                .collect(Collectors.groupingBy(Function.identity(), Collectors.counting()));
        Map<Object, Long> map2 = second.stream()
                .collect(Collectors.groupingBy(Function.identity(), Collectors.counting()));
        return map1.equals(map2);
```

Reply:
this is simpler indeed!